### ISSUE TITLE
Green Belt Land Proposal

### ISSUE THREAD

**Issue by rushughes** [decentraland/proposals#156]
# Preamble
DSP: 
Title: Green Belt Land Proposal
Author: Rus Hughes
Status: Draft
Type: Informational
Created: 2018-04-02

# Abstract
A green belt or greenbelt is a policy and land use designation used in land use planning to retain areas of largely undeveloped, wild, or agricultural land surrounding or neighbouring urban areas. Similar concepts are greenways or green wedges which have a linear character and may run through an urban area instead of around it. In essence, a green belt is an invisible line designating a border around a certain area, preventing development of the area and allowing wildlife to return and be established.

For Decentraland a greenbelt area would be a non-commercial, low poly area developed to create a contrasting, natural, aesthetic to the rest of the Decentraland world with a focus on grass lands, wooded areas, mountains, hills and water feature such as seas, rivers or lakes in order to prevent Decentraland becoming a sequence of cube-like features built to the sky.

# Copyright/public domain
This document is public domain.

# Specification
I propose that unsold LAND is used to form a new district known as Green Belt which can either be managed by Decentraland or existing Decentraland LAND owners or district leaders with the sole goal of creating and maintaining LAND that adhere's to the following tenants:

- Non-commercial - the LAND should be decorative in nature with no commercial plan or intention
- Visually appealing - the LAND should be pleasant to look at and walk around
- Natural - the aesthetic of the LAND should create a nature driven experience, e.g. hills, fields, woods, mountains, stylised parks
- Low poly - the LAND should intentionally aim to be low poly in nature to help give performance boosts to adjacent, owned, LAND
- Low height - in general the LAND should all be low laying, in order to allow Decentraland have areas of clear, unrestricted sky lines that can be viewed.
- Re-imagine famous, natural landmarks from around the world e.g. Grand Canyon, Great Barrier Reef, Mount Everest, Victoria Falls, Northern Lights, Ha Long Bay, The Great Blue Hole, Dead Sea, Eye of the Saraha 
- Usage restricted - agreement of ownership limits usage of the LAND to the green belt idea and it may never be sold or rented in anyway to anyone or used for any other purpose.

# Motivation
In those countries which have them, the stated objectives of green belt policy are to:

- Protect natural or semi-natural environments;
- Improve air quality within urban areas;
- Ensure that urban dwellers have access to countryside, with consequent educational and recreational opportunities; and
- Protect the unique character of rural communities that might otherwise be absorbed by expanding suburbs.

The green belt has many benefits for people:

- Walking, camping, and biking areas close to the cities and towns.
- Contiguous habitat network for wild plants, animals and wildlife.
- Cleaner air and water
- Better land use of areas within the bordering cities.

# Rationale
One of the fears of the Decentraland community is due to the scarcity of the LAND and the small size of the LAND parcels, for the most part people will build dense 10m x 10x x sky high structures on their LAND creating 100% dense urban sprawls making the world unpleasant to experience with no view points.

By creating non-commercial, low laying, low-poly areas of a natural aesthetic it will allow the following:

- Break up density allowing more of the world to become explorable
- Provide a contrast to the sky high tower blocks **everyone** wants to build
- Create city skyline horizon/views - as is Decentraland will not have any interesting views based on consensus of making the most out of all owned spaces.
- Guarantee low-poly areas
- Recreate natural wonders of the world that can be used to create areas that will virally draw in the curious. The Grand Canyon would go viral on Facebook drawing in many Americans. Ha Long Bay would bring in every Vietnamese with a mobile phone in under 24 hours. The Needles would bring in everyone living on the south coast of the UK, the Shinjuku Gyoen National Garden would draw in all of Japan.

# Backwards Compatibility
N/A

# Reference Implementation
N/A

**Comment by rushughes** [comment]
Referencing https://github.com/decentraland/proposals/issues/142 here as this is an alternative idea to an auction

**Comment by greenriverneo** [comment]
YES!  We need this big time.

**Comment by Nyc0las** [comment]
I am totally fond of the idea. I can't wait to look at lakes and walk in Decentraland forest!! If we can have a community (elected by some way?) which can lay out and protect it that would be a good stone to this world.

**Comment by trislit** [comment]
I'm on board with an idea like this. if a voting dapp is successful, perhap determine the closest land owners and allow them votes on how nearby un-owned parcels should be developed (parks/ just trees/decentramon catching area....) to allow for a diverse type of open area. even low poly water can be interesting, especially if random creatures spawn within and interact with the player/environment.

**Comment by 1pocketaces1** [comment]
I agree this is the best proposed use of the land so far!

They should use the same process as district proposals. Have this land set aside for non-commercial, natural landscape, and have people propose natural projects for the space. Then based on community interest in proposals allocate a portion of that land to the proposed project

**Comment by acl-crypto** [comment]
Great project !

**Comment by FadeNot** [comment]
I like the idea. Allocating unowned lands to forests or water (I vote for water) would prevent falling prices by not over saturating a struggling new market. It would also add value to the plots surrounding those areas which mostly are distant from city areas, plazas, roads, and are land locked by other parcels. I imagine they want an auction partially to be able to burn more mana to increase mana value but I think something else could be thought of to do both, increase value of land, mana and beautify DL.

**Comment by lulasky63** [comment]
I love the idea of having protected lands like the USA's National Parks but i don't like the idea of 40% or so of the current city being taken up by them. Maybe something more realistic like 10-15% and since we already have the Genesis City Plaza's that will likely be parks that leaves maybe 20% of the unsold squares needed. I Proposed that we use the unsold lands in the bottom left corner for this idea only so it will be more like real life and we won't have a "woods" section right in the middle of a city.

**Comment by FadeNot** [comment]
I agree no woods in the city. Developers can and will put trees on there property and if the Plazas are going to be like parks there is no need for woodland areas. Woodlands do not increase property value unless you can log them for cash :) Water on the other hand is a different story. In real cities water fronts add a ton of value to property. Many cities are established on lakes, rivers, and ocean fronts. If you have ever lived in a city like this the water adds a ton of value, aesthetics, and adds life to the city.
